config/manager/manager.yaml

```diff
@@ -59,7 +59,7 @@ spec:
           - name: GOOGLE_APPLICATION_CREDENTIALS
             value: /.gce/service-account.json
           - name: PACKET_AUTH_TOKEN
-            value: {{PACKET_AUTH_TOKEN}}
+            value: "{{PACKET_AUTH_TOKEN}}"
```

Comment:
Does this really need to be quoted?

Reply:
It does. kustomize barfs on it if not since it's just a placeholder that the user will need to update.